{"name": "test_create_new_pr", "status": "failed", "statusDetails": {"message": "AssertionError: assert 401 == 201\n +  where 401 = <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DC30>>()\n +    where <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DC30>> = <api.api_responses.Apiresponses object at 0x000001B77FB3DC30>.get_status_code", "trace": "def test_create_new_pr():\n        api = Apirequests(read_base_url())\n        response = api.send_post(pulls, auth=auth, json=params)\n        pr_create_response = Apiresponses(response)\n>       assert pr_create_response.get_status_code() == 201\nE       assert 401 == 201\nE        +  where 401 = <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DC30>>()\nE        +    where <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DC30>> = <api.api_responses.Apiresponses object at 0x000001B77FB3DC30>.get_status_code\n\ntest_creation_of_pullrequest.py:16: AssertionError"}, "start": 1679227578338, "stop": 1679227578595, "uuid": "0645a853-aff3-4ada-ae84-23786e04102a", "historyId": "c291b1acbda530a8b2583d7c04fd5449", "testCaseId": "79adebac2b71d015c39857b386ab34ec", "fullName": "test_creation_of_pullrequest#test_create_new_pr", "labels": [{"name": "suite", "value": "test_creation_of_pullrequest"}, {"name": "host", "value": "AlphaTest"}, {"name": "thread", "value": "2344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_creation_of_pullrequest"}]}